tools/travis/build.sh

```diff
@@ -66,10 +66,12 @@ $ANSIBLE_CMD openwhisk.yml
 
 cd $ROOTDIR
 cat whisk.properties
-TERM=dumb ./gradlew :tests:testLean $GRADLE_PROJS_SKIP
+TERM=dumb ./gradlew tests:testCoverageLean :tests:reportCoverage
```

Comment:
Is the leading colon not needed?

Reply:
It works without that but added now to maintain consistency